frontend/src/pages/projects/screens/detail/overview/serverModels/PlatformSelectSection.tsx

```diff
@@ -18,11 +19,12 @@ const PlatformSelectSection: React.FC = () => (
       </TextContent>
       <Gallery
         hasGutter
-        minWidths={{ default: '100%', lg: 'calc(50% - 1rem / 2)' }}
-        maxWidths={{ default: '100%', lg: 'calc(50% - 1rem / 2)' }}
+        minWidths={{ default: '100%', lg: 'calc(33.33% - 1rem)' }}
+        maxWidths={{ default: '100%', lg: 'calc(33.33% - 1rem)' }}
```

Comment:
This will have to get more dynamic... as discussed in DMs (copying here for unification) 

We shouldn't show the tile if NIMs tile is not installed yet...
![image](https://github.com/user-attachments/assets/83644095-5d10-4122-9df7-5a3e0c2fc68d)

Maybe this is something we can key off of the creation of the ServingRuntime his last PR merged.

Reply:
Updated the logic to display the NIM card conditionally, based on the disableNIMModelServing flag.